checkout/access-checkout-android/src/main/java/com/worldpay/access/checkout/api/discovery/ApiDiscoveryClient.kt

```diff
@@ -18,6 +25,14 @@ internal class ApiDiscoveryClient(
         private const val TAG = "AccessCheckoutDiscoveryClient"
     }
 
+    /**
+     * A method that asynchronously discovers the required API endpoint for the desired service.
```

Comment:
Do not need to say "A method", can begin with "Asynchronously discovers...."

Reply:
updated